seq2science/rules/configuration_generic.smk

```diff
@@ -230,7 +230,8 @@ if "assembly" in samples:
 
     # custom assemblies without provider (for local/remote annotations)
     if "scrna_seq" in get_workflow() and \
-        'kite' in config['quantifier'].get('kallistobus', {}).get('ref', ""):
+       ('kite' in config['quantifier'].get('kallistobus', {}).get('ref', "")) or \
+       (config['quantifier'].get("citeseqcount", "")):
         remote_assemblies = []
         search_assemblies = []
 
```

Comment:
The assembly is not downloaded for ADT based workflow since it is technically not an assembly but a set of custom sequences that is provided in ./tsv/.csv format.